test/type.js

```diff
@@ -45,4 +45,15 @@ describe('type', function() {
     eq(R.type(undefined), 'Undefined');
   });
 
+  it('"Function" if given a function', function() {
+    eq(R.type(() => {}), 'Function');
+  });
+
+  it('"AsyncFunction" if given an async function', function() {
+    eq(R.type(async() => {}), 'AsyncFunction');
+  });
+
+  it('"Generator Function" if given a generator function', function() {
```

Comment:
Typo: extra space in "Generator Function"

Reply:
Fixed with https://github.com/ramda/ramda/pull/2675/commits/b1056f4c1c9509d8417aae1f3c3a40bda135c8f5